components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -226,72 +231,43 @@ export default function UsageBasedBillingConfig({ attributionId }: Props) {
+                                    {usageLimit > 500 ? "Open Source" : "Free"}
+                                </div>
+                                <div className="mt-4 flex space-x-1 text-gray-400 dark:text-gray-500">
+                                    <Check className="m-0.5 w-5 h-5 text-orange-500" />
+                                    <div className="flex flex-col">
+                                        <span className="font-bold text-gray-500 dark:text-gray-400">
+                                            {usageLimit} credits
+                                        </span>
+                                        <span>
+                                            {usageLimit / 10} hours of Standard workspace usage.{" "}
+                                            <a className="gp-link" href="https://www.gitpod.io/pricing#cost-estimator">
+                                                Estimate costs
+                                            </a>
+                                        </span>
+                                    </div>
+                                </div>
+                            </>
+                        )}
                         <div className="bg-gray-100 dark:bg-gray-800 border-t border-gray-200 dark:border-gray-700 -m-4 p-4 mt-8 rounded-b-xl">
                             <div className="uppercase text-sm text-gray-400 dark:text-gray-500">Upgrade Plan</div>
-                            <div className="mt-1 text-xl font-semibold flex-grow text-gray-500 dark:text-gray-400">
-                                {currency === "EUR" ? "€" : "$"}9 / month
-                            </div>
                             <div className="mt-4 flex space-x-1 text-gray-400 dark:text-gray-500">
                                 <Check className="m-0.5 w-5 h-5" />
                                 <div className="flex flex-col">
-                                    <span className="font-bold">1,000 credits</span>
-                                </div>
-                            </div>
-                            <div className="mt-2 flex space-x-1 text-gray-400 dark:text-gray-500">
-                                <Check className="m-0.5 w-5 h-5" />
-                                <div className="flex flex-col">
-                                    <span className="font-bold">Pay-as-you-go after 1,000 credits</span>
                                     <span>
-                                        {currency === "EUR" ? "€" : "$"}0.36 for 10 credits or 1 hour of Standard
-                                        workspace usage, excluding VAT.
+                                        {priceInformation}{" "}
+                                        <a
+                                            className="gp-link"
+                                            href="https://www.gitpod.io/docs/configure/billing/usage-based-billing"
```

Comment:
This link should be updated to point to https://www.gitpod.io/docs/configure/billing

Reply:
👍 I have moved and replaced this link with the estimator link.